.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -50,7 +51,7 @@ jobs:
             name: dns
             clusterName: ${{ parameters.clusterName }}
             ginkgoFocus: '\[sig-network\].DNS.should'
-            ginkgoSkip: 'resolv'
+            ginkgoSkip: 'resolv|256 search'
```

Comment:
I think this based on a different PR?

Reply:
It is based off #2014, but I split it off as it affects multiple pipelines. Did not want it to get stuck within Load Test where it was found.